frontend/cypress/config/blurbs.md

```diff
@@ -0,0 +1,5 @@
+https://github.com/reposense/RepoSense/tree/cypress
```

Comment:
From #2170, it seems like the ultimate goal is for blurbs to be specifiable in the one-stop yaml config implemented in #2192. Is there a reason these PRs are separated?

Is the intention ultimately to merge the two features into a hybrid config of the kind described by prof in https://github.com/reposense/RepoSense/issues/2161#issuecomment-2022776952 ? If so, could you provide some details about this in the PR description?

As it is currently, the blurb markup doesn't actually use any markdown features and the backend updates only seem to forwarding exactly the information in these blurb files into the report json - so there is no real need for backend intervention at all if these blurbs were specified in an identical simple json format to begin with.

Reply:
@gok99 I want the blurbs to be able to support lengthy markdown content using full markdown syntax. So, I don't think they are a good fit for a JSON file. It's easier to edit/preview if they are in a separate .md file.

Example use case: Someone setting up a personal project portfolio page using RepoSense can use blurbs to give extensive details about their contribution to each project. As such a page is likely to have only a small number of projects, it'll have enough space to include lengthy blurbs.

Reply:
My bad, I didn't update the issue with the new direction. I've updated it now!

Reply:
Got it, sorry I had missed the markdown rendering on the frontend - could we also test that some basic markdown features render as expected?

I just have some questions that would affect design considerations:
- Is this feature expected to be used mostly with the one-stop config or be independent of it?
- As @ckcherry23 brings up, do we want blurbs tied only to repositories, or eventually also other groups?

Reply:
The markdown rendering has been used before in #2102, so basic markdown does work.

This feature is currently independent of the one-stop config, since there is no dependency between them. It does have a dependency with the --config flag since it won't be processed otherwise. We can couple them if needed.

Reply:
> The markdown rendering has been used before in https://github.com/reposense/RepoSense/pull/2102, so basic markdown does work.

Can we then also add md inputs to the systemtests to make sure that the backend doesn't mangle things.

Reply:
> Can we then also add md inputs to the systemtests to make sure that the backend doesn't mangle things.

Yes, right now, we are only checking for plain text. We do not need to test if every markdown formatting feature works, but let us add tests to check one markdown formatted blurb (and maybe one HTML formatted blurb) to ensure it is being parsed instead of shown as is. @sopa301